manifests/integreatly-amq-online/amq-online.package.yaml

```diff
@@ -1,4 +1,4 @@
 packageName: rhmi-amq-online
 channels:
 - name: rhmi
-  currentCSV: amqonline.1.3.1
+  currentCSV: amq-online.1.4.0
```

Comment:
This change! Amazing!

Reply:
Just to be clear. I am being serious here. Automation is easier now that this matches the syntax in the filename of the csv.